easybuild/easyconfigs/m/Metal/Metal-2011-03-25-foss-2016a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'Metal'
+version = '2011-03-25'
+
+homepage = 'http://csg.sph.umich.edu/abecasis/Metal/'
+description = """
+Metal - Meta Analysis Helper
+The METAL software is designed to facilitate meta-analysis of large datasets (such as several whole genome scans) in a
+ convenient, rapid and memory efficient manner.
+"""
+
+sources = ['generic-%(namelower)s-%(version)s.tar.gz']
+source_urls = ['http://csg.sph.umich.edu/abecasis/Metal/download/']
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'pic': True}
+
+patches=['Metal.Makefile.patch']
+
+skipsteps=['configure']
+
+buildopts='all'
+
+installopts='INSTALLDIR=%(installdir)s/bin'
```

Comment:
@RvDijk style nitpicking: spaces around `=` in lines above (except for the `INSTALLDIR=` of course)